.golangci.yml

```diff
@@ -0,0 +1,7 @@
+# See https://golangci-lint.run/usage/configuration/#config-file
+
```

Comment:
Introducing this cfg file allows to simplify Taskfile and especially to ensure that everybody using vscode will pick up the correct linting configuration, as long as you have

```
  "go.lintTool": "golangci-lint",
  "go.lintFlags": [
    "--fast"
  ],
```